recipe/bld.bat

```diff
@@ -9,6 +9,7 @@ cmake ^
 	-DLIB_INSTALL_DIR=%LIBRARY_LIB%     ^
 	-DINCLUDE_INSTALL_DIR=%LIBRARY_INC% ^
 	-DSTATIC_CRT=off                    ^
+        -DUSE_SSH=ON                        ^
 	..
 
 ninja install
```

Comment:
Can this speed up the building time https://github.com/anaconda-distribution/distro-incubator/blob/50336807b4d23581a442dc4ef7677ba5abfd278a/skel/build.sh#L21-L35 ?
The maintainers use cores too https://github.com/libgit2/libgit2/blob/fbea439d4b6fc91c6b619d01b85ab3b7746e4c19/ci/build.sh#L73